improver_tests/cube_combiner/test_Combine.py

```diff
@@ -146,5 +161,5 @@ def test_minimum_realizations_exceptions(
 
 def test_empty_cubelist():
     """Ensure supplying an empty CubeList raises an error"""
-    with pytest.raises(TypeError, match="A cube is needed to be combined."):
+    with pytest.raises(ValueError, match="A cube is needed to be combined."):
```

Comment:
Ditch this if you remove this test from the plugin and rely on the as_cubelist function to raise a suitable exception.

Reply:
Sure, I assumed to maintain non-functional changes where it made sense to do so.
As you have now confirmed, happy to remove (as with anything where I have avoided changing things) 👍